src/main/java/com/afornalik/ox/controller/SkirmishController.java

```diff
@@ -0,0 +1,58 @@
+package com.afornalik.ox.controller;
+
+import com.afornalik.ox.model.board.Board;
+import com.afornalik.ox.model.board.FieldStatus;
+import com.afornalik.ox.model.board.OutOfBoardException;
+import com.afornalik.ox.model.player.PlayerContainer;
+import com.afornalik.ox.view.UIOperations;
+
+class SkirmishController {
+
+    private final UIOperations uiConsole;
+    private final Board board;
+    private final PlayerContainer playerContainer;
+
+    SkirmishController(UIOperations uiConsole, Board board, PlayerContainer playerContainer) {
+        this.uiConsole = uiConsole;
+        this.board = board;
+        this.playerContainer = playerContainer;
+    }
+
+    Board makeMove(FieldStatus fieldStatus) {
+        int index;
+        try {
+            FieldStatus checkStatus = FieldStatus.EMPTY;
+            do {
+                index = uiConsole.readNumber();
+                try {
+                    checkStatus = board.receiveBoardField(index - 1);
+                } catch (OutOfBoardException e2) {
+                   // uiConsole.print(e2.getMessage());
+                }
+                if (checkStatus != FieldStatus.EMPTY){
+                    uiConsole.print("\nField already taken choose different number ");
+                }
+            } while (checkStatus != FieldStatus.EMPTY);
+            board.insertBoardField(index - 1, fieldStatus);
+        } catch (OutOfBoardException e) {
+            uiConsole.print(e.getMessage());
+        }
+        return board;
+    }
+
+    Board makeTurn() {
+        uiConsole.drawBoard();
+        if (board.receiveNumberOfEmptyFields() == 0) {
```

Comment:
One turn is one player move. Here you have two, which means you have to duplicate a lot of code and check if there are still some empty fields on board twice.
Also, board.receiveNumberOfEmptyFields() == 0 isn't very easy to understand, I had to go to board class to make sure I understand what you are checking here. Maybe move it to separate method (like board.isFull or something)? Knowing that 0 = no more fields to put symbols on is board responsibility after all.

Reply:
Refactored